server/index.d.ts

```diff
@@ -2,11 +2,19 @@ import BaseServer, { Logger, Reporter, BaseServerOptions } from '../base-server'
 
 export type ServerOptions = BaseServerOptions & {
```

Comment:
I think we need also change `./test-server/` options.

Reply:
I "analyzed" usages of `TestServer` and it seems, that we either want to provide custom reporter or use `'human'` logger with default reporter. So there is no need to add logger parameter, just remove `'human'` from reporter and make it default will suffice?